src/pages/components/tabs-extended.mdx

```diff
@@ -41,6 +42,7 @@ The tab labels may wrap up to two lines then truncated.
 
 On mobile, Tabs extended switches to an accordion for easier browsing.
 
```

Comment:
@RichKummer I don't think we need the `Behavior` heading here. But I would add captions to underneath each image.

Example of the tabs extended component, on desktop

Example of the tabs extended component, on mobile